ansible/roles/prometheus-fed/templates/prometheus.yml

```diff
@@ -0,0 +1,45 @@
+global:
+  scrape_interval: 1m
+  scrape_timeout: 10s
+  evaluation_interval: 1m
+  # Clustername is defined for multi swarm cluster
+  # can name first cluster as DS1 and 2nd as DS2 ... DSn
+  # all metrics will have label as cluster: < value >
+  # This value won't be accessible from GUI but for external tools like
+  # federation or alertmanager
+  external_labels:
+    federation: fed1
```

Comment:
change the variable to cluster, so that we can have the same external label across all proms

Reply:
as this is just doing a aggregation, we dont have to keep it as a cluster.